packages/flutter_test/lib/src/matchers.dart

```diff
@@ -280,6 +280,9 @@ Matcher coversSameAreaAs(Path expectedPath, { @required Rect areaToCompare, int
 /// await expectLater(imageFuture, matchesGoldenFile('save.png'));
 /// ```
 ///
+/// Golden image files can be created by running `flutter test --update-goldens`
```

Comment:
maybe say: "... can be created or updated by ..." to be slightly more specific?

Reply:
I was trying to be less wordy since it's creating over the existing file. =) (It also meant that I wouldn't have to split the command onto the next line.)

But okay, I guess it's not necessarily obvious whether existing files can be overwritten.  Updated.